.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## What changes were made
+
+<!-- Describe the changes you did -->
+
+## How to test or check results
+
+<!-- URL or instructions -->
+
+## Checks
+- [ ] All content model changes are done through [scripted migrations](../readme.md#scripted-migrations)
+- [ ] Appointed PR reviewers. At least 1 [code owner](./codeowners) should review
+- [ ] Notify reviewers (e.g. send a message in the slack channel)
```

Comment:
@lsarni thoughts on this? Think for most PRs you don't send a message right?
I use GH notifications so I prefer those but I think Sjoerd & Frank don't... :thought_balloon:

Reply:
I've removed this check. I sometimes just check PRs. And I think it's the PR owner's responsibility to make sure the reviewer knows he/she is requested a review and the preferred way may defer per reviewer.